components/atoms/TransparentButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Button, type ButtonProps, Text } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+import { RADIUS_BUTTON } from '~/constants/Constants'
+import { useAppFonts } from '~/hooks/useAppFonts'
+
+type Props = {
+  title: string
+  onPress?: () => void
+} & ButtonProps
+const TransparentButton = (props: Props): React.ReactElement => {
+  const { fonts } = useAppFonts()
+  const colors = getColors(useColorScheme())
+  return (
+    <Button
+      onPress={props.onPress}
+      bg="$colorTransparent"
```

Comment:
You should use the available color prepared, to achieve uniformity in color

Reply:
It's transparent so I had to use it like that